spring/src/main/java/com/gcinterceptor/spring/SpringGcInterceptorConfiguration.java

```diff
@@ -21,7 +21,7 @@ public SpringGcInterceptor springGcInterceptor() {
     }
 
     @Bean
```

Comment:
This returned could be a singleton. We don't want to loop over the memory pools for every request.